user_guide_src/source/models/model.rst

```diff
@@ -144,9 +144,9 @@ can maintain a "recycle bin" of objects that can be restored, or even simply pre
 part of a security trail. If true, the find* methods will only return non-deleted rows, unless
 the withDeleted() method is called prior to calling the find* method.
 
-This requires a DATETIME field to be present in the table for storing state. The default field
-name is  ``deleted_at`` however this name can be configured to any name of your choice by using
-$deletedField property.
+This requires requires either a DATETIME or INTEGER field in the database as per the model's
```

Comment:
Double `requires` here. Only need one :)

Reply:
Is that a risk you're willing to take?

Reply:
@MGatner I'm not sure I follow - but yes having it say `This requires either` instead of `This requires requires either` is a risk I'm willing to take. :)

Reply:
:) just joking. It's all set, if everything looks good to you!